packages/devtools_app/lib/src/shared/offline_screen.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
git diff didn't pick up that this was a renamed file. snapshot_screen ==> offline_screen

I will comment on the changed parts of this code